ast_generic.opam

```diff
@@ -1,5 +1,5 @@
+# This file is generated by dune, edit dune-project instead
```

Comment:
Is this file is generated by Dune, do we need to keep it?

Reply:
I kept it because it lists dependencies. If the convention is that all the project dependencies are explicitly listed in `semgrep.opam`, then we don't need other opam files.